lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
+
+  attr_reader :country,
+              :players
+
+  def initialize(country)
+    @country = country
+    @players = []
+
+  end
+
+  def eliminated?
+    false
+  end
+
+  def add_player(player)
+    @players << player
+  end
+
+  def players_by_position(player_instance)
```

Comment:
This is sooo close!  Be careful on line 21 - you are setting the variable `players_in_position` to the actual position, and then on line 23, I think you want to call `@players` to iterate through the list of players.  I think using pry to help debug would be _really_ beneficial here to challenge your assumptions.